.jupyter-releaser.toml

```diff
@@ -0,0 +1,15 @@
+[hooks]
```

Comment:
Maybe this config could be moved to `pyproject.toml` for consistency with the other Jupyter projects?

Reply:
I would also like to split out Hatch configuration into `hatch.toml` later on.

Reply:
Sure. Sometimes having many configuration files tends to be quite overwhelming. Also some other tools like `check-manifest`, `black` or `mypy` are often configured in `pyproject.toml`.

Reply:
Thank you all for your input. I want us to experiment with this for now; I think having tool-specific configuration files is easier than scanning each line of `pyproject.toml` to find the relevant lines. We can reverse this easily if it turns out that this was not a good change.